### DISCUSSION TITLE
Estimate Shipping & Tax in Cart

### DISCUSSION THREAD

**Discussion by octoxan** [solidusio/solidus#4591] [New Features or Ideas]
**Is your feature request related to a problem? Please describe.**
In most e-commerce frameworks there is a box on the cart page to input you postal code / country and show a shipping + tax estimate, usually allowing you to pre-select a shipping option. This is in Magento 1 & 2, Woo Commerce, etc. Building our first Solidus site and our designer included this box in the design as we were used to it being in every platform so far. Would be a great addition to Solidus, I think.

**Describe the solution you'd like**
A new box at the cart page that allows a user to input general location information to select a shipping method and preview any tax, before having to go through the checkout.

**Comment by seandawes** [comment]
This could be done in conjunction with the existing Easypost extension out there potentially which is already supported by Solidus

https://github.com/solidusio-contrib/solidus_easypost

**Comment by jarednorman** [comment]
Certainly would be useful, but I'd be wary about making sure we draw the line correctly between what's in core and what might be better offered by extensions like the one Sean linked.
